style.css

```diff
@@ -64,3 +70,213 @@ a {
   margin-bottom: 30px;
   padding-bottom: 20px;
 }
+
+.recent-works {
+  color: #172b4d;
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 16px;
+}
+
+.normal-button {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 62px;
+}
+
+hr {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 60px;
+  background: #36b37e;
+  border-radius: 24px;
+  width: 48px;
+  height: 4px;
+}
+
+.recent-work__items {
+  color: #3a4a42;
+  text-align: center;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  margin: 16px;
+}
+
+.recent-work__items-languages {
+  display: flex;
+  flex-wrap: wrap;
+  align-self: center;
+  list-style: none;
+  justify-content: center;
+  column-gap: 8px;
+  margin-top: 30px;
+  padding: 11px;
+}
+
+.recent-work__items-languages > li {
+  padding: 8px 12px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  margin-bottom: 8px;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+.see-projects {
+  text-decoration: none;
+  color: #fff;
+  background-color: #36b37f;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  width: 122px;
+  margin-bottom: 24px;
+}
+
+.profile {
+  display: grid;
+}
+
+.abouttop {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  padding-top: 30px;
+}
+
+.profile-me {
+  color: #172b4d;
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 16px;
+}
+
+.profile-intro {
+  color: #172b4d;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  padding: 5px;
+}
+
+.recent-work {
+  display: grid;
+}
+
+article {
```

Comment:
Kindly use CSS classes for styling the HTML elements. As CSS classes are reusable It's recommended as well. 👍🏻